docs/learn-sapio/src/ch01-01-installation.md

```diff
@@ -21,9 +21,11 @@ rustup target add wasm32-unknown-unknown
 > and then load the following before compiling
 > ```bash
 > export PATH="/opt/homebrew/opt/llvm/bin:$PATH"
+> # for older homebrew installs
+> # export PATH="/usr/local/opt/llvm/bin:$PATH"
```

Comment:
Does this work given the CC/AR overrides below?

Reply:
Ah, good point.

I actually just made /opt/homebrew/opt/ and then symlinked /usr/local/opt/llvm to /opt/homebrew/opt/llvm.